sale_order_team_from_product/readme/CONTEXT.md

```
@@ -0,0 +1,2 @@
+Sometimes you need to set a product specific sales team in quotation.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)